cumulus/conf/default_config.json

```diff
@@ -11,6 +11,9 @@
      "moabReader": {
             "pluginPath": "/test"
     },
-    "amis": {}
+    "amis": {},
+    "taskflow": {
+        "path": ["tests/integration/taskflow"]
```

Comment:
copying the default into my config.json,  I end up getting task_runner as a task - is that expected behavior?